lib/transport/selenium-webdriver/browserstack.js

```diff
@@ -70,6 +70,12 @@ class Browserstack extends BaseDriver {
       this.settings.desiredCapabilities['bstack:options'].buildName = 'nightwatch-test-build';
     }
 
+    if(this.settings.local_setup) {
+      this.settings.desiredCapabilities['bstack:options'].local = true;
```

Comment:
What about jsonwp ?

Reply:
Handling of jsonwp is missing

Reply:
Nightwatch translating jsonwp to w3c internally. Handling not required